semantic_release/cli.py

```diff
@@ -204,6 +197,35 @@ def publish(**kwargs):
             click.echo(
                 click.style('Missing token: cannot post changelog', 'red'), err=True)
 
+        # Get config options for uploads
+        upload_pypi = config.getboolean('semantic_release', 'upload_to_pypi')
+        upload_release = config.getboolean('semantic_release', 'upload_to_release')
+        if upload_pypi or upload_release:
+            path = config.get('semantic_release', 'dist_path') or 'dist'
+            remove_dist = config.getboolean('semantic_release', 'remove_dist')
+
+            click.echo('Building distributions')
+            if remove_dist:
+                # Remove old distributions before building
+                remove_dists(path)
+            build_dists()
+
+            if upload_pypi:
+                click.echo('Uploading to PyPI')
+                upload_to_pypi(
```

Comment:
Please move this back before posting the changelog. The posting of changelog will send notification to watchers of the repo.

Reply:
Thing is, the release needs to be created first to be able to upload the assets to it. I could post the changelog in between the PyPI upload and the GitHub upload?